### ISSUE TITLE
get an error when updating to latest version

### ISSUE THREAD

**Issue by haarperr** [TheMrDeivid/ESX_shops_with_bars#8]
https://imgur.com/a/DduF32e

**Comment by TheMrDeivid** [comment]
> https://imgur.com/a/DduF32e

Hey, i don't know what could be causing that, everything is working fine on my end, did you try clear your cache and restart the entire server sir? 

Check if you have all the images for all the items

**Comment by Andromeda-stack** [comment]
Having the same issue. Nothing shows at the shops expect blips. Sql files imported. everything is setup correctly. getting the nil(?) error

**Comment by TheMrDeivid** [comment]
> Having the same issue. Nothing shows at the shops expect blips. Sql files imported. everything is setup correctly. getting the nil(?) error

I did some updates now! I haved tested everything! looks like it was a missed placed item in the sql, and some other items in the resource.lua, everything working on my end!

**Comment by Andromeda-stack** [comment]
Ill test it tomorrow on my end, and ill let you know! Thanks

**Comment by TheMrDeivid** [comment]
> Ill test it tomorrow on my end, and ill let you know! Thanks

Sure! Let me know how it goes!
Make sure you use also the esx_basicneeds that comes with the download!

**Comment by Duqino** [comment]
Hi.

I just installed the lastest version and i have allmost the same problem. I did upload your version of esx_basicneeds

SCRIPT ERROR: @esx_shops/server/main.lua:68: attempt to index a nil value (field '?')
> handler (@esx_shops/server/main.lua:35)

Any idea why its showing this in console?

**Comment by wanderrer** [comment]
> 
> 
> Hi.
> 
> I just installed the lastest version and i have allmost the same problem. I did upload your version of esx_basicneeds
> 
> SCRIPT ERROR: @esx_shops/server/main.lua:68: attempt to index a nil value (field '?')
> 
> > handler (@esx_shops/server/main.lua:35)
> 
> Any idea why its showing this in console?

Yes, it's ESX (es_extended), they introduced weights and fucked everything (most of the scrips based on buy/sell/gather/pick) everything must be modified to accept the new freaking weight ..... it's so frustrating... for the past weeks I'm still working on my server....Jesus Christ....
